python/pyspark/mllib/util.py

```diff
@@ -169,6 +169,25 @@ def loadLabeledPoints(sc, path, minPartitions=None):
         minPartitions = minPartitions or min(sc.defaultParallelism, 2)
         return callMLlibFunc("loadLabeledPoints", sc, path, minPartitions)
 
+    @staticmethod
+    def appendBias(data):
+        """
+        Returns a new vector with `1.0` (bias) appended to
+        the end of the input vector.
+        """
+        vec = _convert_to_vector(data)
+        if isinstance(vec, Vector):
+            vec = vec.toArray()
```

Comment:
This is going to make sparse vectors become dense.  It should treat SparseVector and scipy.sparse column matrices specially so data size does not blow up.  There are probably numpy or scipy methods which could help with that.